public_website/views.py

```diff
@@ -937,8 +990,8 @@ def getStudentClassStat(self, params = None):
         for tple in class_tuples:
             student_class = tple[0]
             student_count = tple[1]
-            if student_count < 20 : # 10 is arbitrary here. Change it as per the requirements. 
-                # ignore if count of student from this class is less than 10 
+            if student_count < 1 : # 20 is arbitrary here. Change it as per the requirements. 
```

Comment:
The comment needs to be updated! :)

Reply:
Sounds good. Maybe we can move this option to `settings.py` too at some point to avoid too many PRs? We could set a default like `max_allowed = settings.get('MAX_STUDENT_COUNT') or 5`